mitx_residential_etl.py

```diff
@@ -138,14 +138,16 @@ def mysql_query(course_ids):
     connection = engine.connect()
     for course_id in course_ids:
         for key, value in query_dict.items():
-            query_text = text(value)
+            query_text = text(value['command'])
+            fieldnames = value['fieldnames']
             query = connection.execute(query_text, course_id=course_id.decode('utf8'))
-            write_csv(query, key)
+            write_csv(query, key, fieldnames)
 
 
-def write_csv(query, key):
+def write_csv(query, key, fieldnames):
     with open(daily_folder + str(key) + '.csv', 'a+', encoding="utf-8") as f:
         writer = csv.writer(f)
```

Comment:
You can set this as a DictWriter and then instead of `writerow(fieldnames)` the next line would be `writer.writeheader()`. That should also allow you to leave the query as `select *` and then specify the fieldnames that you want written to the CSV when you instantiate the writer. https://docs.python.org/2.7/library/csv.html#csv.DictWriter